devApps/VanillaJSTestApp/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "msal-js-devapp",
+  "version": "1.1.0",
```

Comment:
I'll make it 0.1.0

---
In reply to: [114031943](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/27#discussion_r114031943) [](ancestors = 114031943)